examples/maven/pom.xml

```diff
@@ -27,7 +27,6 @@
                     <diktatConfigFile>diktat-analysis.yml</diktatConfigFile>
                     <inputs>
                         <input>${project.basedir}/src/main/kotlin</input>
-                        <input>${project.basedir}/src/test/kotlin</input>
```

Comment:
suggestion to leave in examples `/test` as input value

Reply:
Done